config.yaml

```diff
@@ -0,0 +1,5 @@
+index_nucleotides: 3
+heap_space: 3
```

Comment:
Yeah so when i tried this on i run into issues where the travis build run out of memory. I read up on this a found that there is actually only 3g available for each build so this is why its reduced. Since its a config it is however easy to change this for larger dataset when running on uppmax.

Reply:
It's strange that it worked before. Either way, then I think it is better to change the .travis file since it only has to do with that test. In this scenario we could also put a comment explaining it as well without confusing our future selfs with why it is set to 3.